src/pages/MerchantVoucherDashboardV2/styles.module.scss

```diff
@@ -0,0 +1,287 @@
+
+  @media only screen and (max-width: 960px) {
+    flex-direction: column;
+    padding: 40px 28px;
+  }
+
+  @media only screen and (max-width: 414px) {
+    padding: 20px 16px;
+  }
+
+  .headerTitle {
+    color: white;
+    font-size: 32px;
+    font-weight: 700;
+    letter-spacing: -0.01em;
+    line-height: 44px;
+    margin-bottom: 16px;
+  }
+
+  .headerSubtitle {
+    color: white;
+    font-size: 18px;
+    letter-spacing: 0.05em;
+    line-height: 25px;
+    text-transform: uppercase;
+  }
+
+  .actionButtons {
+    > :first-child {
+      margin-right: 24px;
+    }
+
+    @media only screen and (max-width: 960px) {
+      margin-top: 32px;
+    }
+
+    .loadingButton {
+      background-color: #ffffff;
+      border-radius: 50px;
+      color: #a8192e;
+      font-size: 16px;
+      font-weight: 700;
+      letter-spacing: 2px;
+      padding: 16px 50px;
+      text-transform: uppercase;
+
+      .loadingButtonIcon {
+        display: flex;
+        height: 24px;
+        width: 24px;
+      }
+
+      .loadingText {
+        font-family: 'Open Sans';
+        margin-left: 12px;
+      }
+    }
+  }
+}
+
+.stats {
+  background-color: $light-red;
+  display: flex;
+  justify-content: space-between;
+  padding: 48px 120px;
+
+  @media only screen and (max-width: 960px) {
+    padding: 24px 28px;
+  }
+
+  @media only screen and (max-width: 414px) {
+    padding: 12px 16px;
+  }
+
+  @media only screen and (max-width: 960px) {
+    flex-direction: column;
+
+    > div {
+      margin-bottom: 32px;
+    }
+
+    :last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  .statsTitle {
+    margin-bottom: 8px;
+    font-size: 16px;
+    text-transform: uppercase;
+  }
+
+  .statsSubtitle {
+    font-size: 24px;
+    font-weight: 700;
+    letter-spacing: -0.01em;
+  }
+}
+
+.banner {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  padding: 22px 32px;
+}
+
+.successfulSaveBanner {
+  background-color: #f8ba17;
+
+  .successfulSaveIconContainer {
+    color: #a8192e;
+    margin-right: 8px;
+  }
+}
+
+.saveErrorBanner {
+  background-color: #a8192e;
+  color: #ffffff;
+
+  .errorIconContainer {
+    color: #ffffff;
+    margin-right: 8px;
+  }
+}
+
+.contentContainer {
+  padding: 32px 120px;
+
+  @media only screen and (max-width: 960px) {
+    padding: 16px 28px;
+  }
+
+  @media only screen and (max-width: 414px) {
+    padding: 16px 0;
+  }
+}
+
+.filterSection {
+  display: flex;
+  margin-bottom: 28px;
+  width: 100%;
+}
+
+.searchBar {
+  border-radius: 5px;
+  border: 1px solid #c4c4c4;
+  display: flex;
+  flex-grow: 1;
+  padding: 16px;
+
+  .searchIcon {
+    color: $light-grey;
+    height: 18px;
+    margin-right: 12px;
+    pointer-events: none;
+    width: 18px;
+  }
+
+  .searchText {
+    width: 100%;
+  }
+}
+
+.filterGamContainer {
+  align-items: center;
+  display: flex;
+  margin-left: 60px;
+
+  @media only screen and (max-width: 960px) {
+    margin-left: 8px;
+  }
+
+  .checkbox {
```

Comment:
I try to avoid nested classes because it's usually misused and leads to a lot of CSS duplication